packages/web-components/fast-foundation/src/text-field/text-field.ts

```diff
@@ -251,7 +252,27 @@ export class TextField extends FormAssociated<HTMLInputElement> {
         if (this.control && this.control.value) {
```

Comment:
```suggestion
        if (this.control) {
```

The reason for this is that `value` could be the empty string after backspacing, which is a legitimate change. So, that needs to be reported.

Reply:
added this suggestion